Tasks/XamarinAndroid/task.json

```diff
@@ -54,35 +53,48 @@
             "name": "outputDir",
             "type": "string",
             "label": "Output Directory",
-            "defaultValue": "$(Agent.BuildDirectory)\\bin\\Release",
+            "defaultValue": "",
```

Comment:
If we are removing the default value, it would be nice to provide helpMarkDown for this field.  Something like "Optionally provide the output directory for the build.  Otherwise, build output will be located in the XYZ directory.  Example: $(Agent.BuildDirectory)/bin/Release"